apps/frontend/src/app/Components/FieldDisplay.tsx

```diff
@@ -125,6 +126,11 @@ export function NodeFieldDisplay({
             {' '}
             {diff > 0 ? '+' : ''}
             {valueString(diff, node.info.unit, node.info.fixed)}
+            {(node.info.unit != "%" && oldValue != 0) ? (
+              " (" + pctDiff + ")"
```

Comment:
Can you use template literals please?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Done in [582d259](https://github.com/frzyc/genshin-optimizer/pull/1288/commits/582d259fc51f12597ff75674a5c0fbfe5f7b83f6)